ddp-workspace/projects/ddp-sdk/src/lib/directives/lazyLoadResources.directive.spec.ts

```diff
@@ -0,0 +1,74 @@
+/* tslint:disable:max-classes-per-file */
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { Component, DebugElement } from '@angular/core';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { MatCheckboxModule } from '@angular/material/checkbox';
+import { By } from '@angular/platform-browser';
+import { LazyLoadResourcesDirective, WindowRef } from 'ddp-sdk';
+import { FormsModule } from '@angular/forms';
+
+fdescribe('LazyLoadResourcesDirective', () => {
```

Comment:
please remove 'f'.
`fdescribe` => `describe`

Reply:
thanks! strange there is no pre-push hook for that. I guess we need to add it at some point